tour_model_eval/get_scores.py

```diff
@@ -0,0 +1,42 @@
+import pandas as pd
+from pandas.testing import assert_frame_equal
+import label_processing as label_pro
+from sklearn import metrics
+
+
+
+# compare the trip orders in bin_trips with those in filter_trips above cutoff
+def compare_trip_orders(bins,bin_trips,filter_trips):
+    bin_trips_ts = pd.DataFrame(data=[trip["data"]["start_ts"] for trip in bin_trips])
+    bin_ls = []
+    for bin in bins:
+        for index in bin:
+            bin_ls.append(index)
```

Comment:
why do you need to iterate like this? Why not just use the bins directly? Is it to allow us to `flatten`?

Reply:
resolved